src/Pornsearch.js

```diff
@@ -1,23 +1,22 @@
-'use strict';
+import Axios from 'axios';
+import Cheerio from 'cheerio';
 
-const AbstractModule = require('./Core/AbstractModule');
-const Axios          = require('axios');
-const Cheerio        = require('cheerio');
-const FS             = require('fs');
-const Path           = require('path');
+import pornhub from './Modules/Pornhub';
+import redtube from './Modules/Redtube';
+import sex from './Modules/Sex';
+import xvideos from './Modules/Xvideos';
```

Comment:
I think it's better to have an file in /core exporting all modules

Reply:
Working on it.